app/policies/application_policy.rb

```diff
@@ -32,4 +32,17 @@ def account_id
   def account_id_was
     nil
   end
+
+  class Scope
+    attr_reader :token, :scope
```

Comment:
Ported from Augury.  But... the pattern in Feeder seems to be calling it a `token` vs a `user`.  So stuck with that here.  (It's the same `Prx::Auth::Token` so doesn't really matter).

Reply:
It seems they already are?  I've actually been using `current_user` in other places (outside of the Policies).